logarithms_refreshed.Rmd

```diff
@@ -130,12 +130,28 @@ np.log10(np.sqrt(10))
 ```
 
 It is a bit difficult to think about what $10^0$ might mean, but the rule is
-that $10^0$ is equal to 1, and therefore log10 of 1 is 0:
+that $10^0$ is equal to 1. This is the case because the way we calculate divisions like $10^4/10^2$ is by subtracting the bottom exponent from the top exponent e.g. $10^4/10^2 = 10^{4-2} = 10^2$. 
```

Comment:
This is nice - but maybe this should go at the end when they know about the multiply / add stuff?    Can we just say something like we'll justify this later?  Then put this staff after multiply / add stuff.

I guess we'd also need to say expliclity that dividing is subtracting exponents and give an example like:

```
\frac{10^3}{10^2} = 
\frac{10 * 10 * 10}{10 * 10} = 10 
```

and then your 10^0 example?

Reply:
I've added a section after the multiply/add section.  I haven't done the subtracting/dividing section yet thought.

Also, sent some suggestions for the logistic regression page itself.